modules/op-about-tasks.adoc

```diff
@@ -7,7 +7,7 @@
 
 _Tasks_ are the building blocks of a Pipeline and consist of sequentially executed Steps. Tasks are reusable and can be used in multiple Pipelines.
 
-_Steps_ are a series of commands that achieve a specific goal, such as building an image. Every Task runs as a pod and each Step runs in its own container within the same pod. Because Steps run within the same pod, they have access to the same volumes for caching files, ConfigMaps, and Secrets.
+_Steps_ are a series of commands that achieve a specific goal, such as building an image. Every Task runs as a pod, and each Step runs as a container within that pod. Because Steps run within the same pod, they can access the same volumes for caching files, ConfigMaps, and Secrets.
```

Comment:
It's technically out of scope for this PR, but if you're inclined to make an extraneous update:
s/ConfigMaps/config maps
s/Secrets/secrets